ports/atmel-samd/boards/shirtty/mpconfigboard.mk

```diff
@@ -0,0 +1,15 @@
+LD_FILE = boards/samd21x18-bootloader.ld
+USB_VID = 0x239A
+USB_PID = 0x801F
```

Comment:
USB VID and PID are something you cannot borrow from another board. So don't copy and paste these values! If you don't have a VID/PID pair, open up an issue in CircuitPython repository and request one, we will assist in the process of assigning a PID for use with CircuitPython.

https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/customizing-the-board-files#mpconfigboard-dot-mk-5-25

Reply:
Oops - Yes I am aware and fully understand. I wanted to get this PR opened ASAP after the conference. I will RTFM the guide on how to add a new board and update this. Changing the title to make it clear it's not ready to merge.

Reply:
sounds good - thanks :)